collection.js

```diff
@@ -42,13 +42,14 @@ class Library {
   render() {
     this.list.innerHTML = this.collection
       .map(
-        (book) => `<li>
+        (book) => `<li class="bookRow">
       <article>
           <p id="title">"${book.title}" by ${book.author}</p>
           <button class="remove">Remove</button>
       </article>
       </li>`,
-      ).join('');
+      )
+      .join('');
 
     const remove = Array.from(document.getElementsByClassName('remove'));
```

Comment:
- [OPTIONAL] It would be a good UI design to display a message anytime a book is added.